### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by pinology**
homework 3

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         

Clones added
============
- Src/BootCamp.Chapter/Lesson3.cs  1
- Src/BootCamp.Chapter/Checks.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4867708&bid=16140706)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4867708)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4867708)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4867708&bid=16140706)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4867708&bid=16140706)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo()
+        {
+            //run twice
+            for (int i = 0; i < 2; i++)
+            {
+                string firstName = PromptString("Input First Name: ");
```
Comment (diverges) [inline Src/BootCamp.Chapter/Lesson3.cs]: This can probably be its own function :)

**Pull request closed without merging**